templates/common/infra/bicep/core/storage/storage-account.bicep

```diff
@@ -1,3 +1,4 @@
+metadata description = 'Creates a Storage Account.'
```

Comment:
```suggestion
metadata description = 'Creates an Azure storage account.'
```

Reply:
Refer to https://learn.microsoft.com/en-us/azure/storage/common/storage-account-overview